### PR TITLE
Remove more //base dependencies

### PR DESCRIPTION
**Pull request by abarth**
WTF is now //base clean.

### CODE REVIEW

**Comment by abarth** [comment]
@jamesr

File: sky/engine/core/rendering/RenderView.cpp [hunk 1]
```diff
@@ -60,7 +59,6 @@ bool RenderView::hitTest(const HitTestRequest& request, HitTestResult& result)
 
 bool RenderView::hitTest(const HitTestRequest& request, const HitTestLocation& location, HitTestResult& result)
 {
-    TRACE_EVENT0("blink", "RenderView::hitTest");
```
Comment (jamesr) [inline sky/engine/core/rendering/RenderView.cpp]: you have another TRACE_EVENT in glue, right?

wait you have a hitTest function in C++?

**Comment by jamesr** [comment]
lgtm

File: sky/engine/core/rendering/RenderView.cpp (hunk 1 above)
Comment (abarth) [inline sky/engine/core/rendering/RenderView.cpp]: > you have another TRACE_EVENT in glue, right?

Yes.

> wait you have a hitTest function in C++?

This code is dead.  We just haven't deleted it yet.  :)

**Pull request merged**